hack/scripts/patch-remote/patch_remote.py

```diff
@@ -76,6 +76,7 @@ def __init__(self, conf_file, private_key, targets, verbose):
         self._arch = self._config.get("NUCLIO_ARCH", "amd64")
         self._namespace = self._config.get("NAMESPACE", "nuclio")
         self._private_key = private_key
+        self._os = self._config.get("NUCLIO_OS", "linux")
```

Comment:
Add this env var to `patch_env_template.yml` with a proper comment, and without a value.